library/n2_protocol.cpp

```diff
@@ -137,6 +144,15 @@ void n2_response_info_free(struct n2_response_info *resp_info) {
 	delete resp_info;
 }
 
+struct n2_response_info *n2_response_info_create_timed_out(struct dnet_cmd *cmd, struct n2_repliers *repliers) {
```

Comment:
Not only "timeout" may happen with transaction. There is also code in `dnet_trans_clean_list()` that is called when a connection is broken.

Reply:
Reworked on `n2_response_info_create_from_error(t, repliers, err)`.

In `dnet_trans_clean_list` replier is called directly, without conversion to `n2_response_info`. Such a conversion was need only in `dnet_trans_check_stall`, because call is done in another place (in `dnet_trans_enqueue_responses_on_timed_out`).